app/src/main/java/org/liberty/android/fantastischmemo/dao/MCDao.java

```diff
@@ -0,0 +1,27 @@
+package org.liberty.android.fantastischmemo.dao;
+
+
+import org.liberty.android.fantastischmemo.common.AnyMemoDBOpenHelper;
+import org.liberty.android.fantastischmemo.entity.MultipleChoiceCard;
+
+import java.util.List;
+
+public interface MCDao extends HelperDao<MultipleChoiceCard, Integer> {
```

Comment:
Change MCDao to MultipleChoiceDao
your entitly is called MultipleChoiceCard so stay consistent